Formula/lib/libmsquic.rb

```diff
@@ -26,7 +26,7 @@ class Libmsquic < Formula
 
   def install
     args = %w[
-      -DQUIC_TLS=openssl3
+      -DQUIC_TLS_LIB=quictls
```

Comment:
Probably should be

```suggestion
      -DQUIC_TLS_LIB=openssl
```

Reply:
`openssl` is on the main, but not yet for the version 2.5.

```
if(QUIC_TLS_LIB STREQUAL "quictls")
    add_library(OpenSSL INTERFACE)

    include(FetchContent)

    FetchContent_Declare(
        OpenSSLQuic
        SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/submodules
        CMAKE_ARGS "-DQUIC_USE_SYSTEM_LIBCRYPTO=${QUIC_USE_SYSTEM_LIBCRYPTO}"
    )
    FetchContent_MakeAvailable(OpenSSLQuic)

    target_link_libraries(OpenSSL
        INTERFACE
        OpenSSLQuic::OpenSSLQuic
    )
endif()
```

Reply:
```
CMake Error at src/platform/CMakeLists.txt:59 (message):
  TLS Provider not configured


-- Configuring incomplete, errors occurred!
/opt/homebrew/Library/Homebrew/ignorable.rb:30:in 'block in Object#raise
```

Reply:
Yea, maybe we should wait? I kinda want to avoid setting `quictls` here, since it vendors its own version of `quictls`.

Reply:
I agree that we should wait for openssl, until then we can stick with 2.4.x versions on MsQuic here. My preference is also openssl on the package.